.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -0,0 +1,36 @@
+name: 📝 Documentation
+description: Contribute to improving Scribe-Android's documentation!
+labels: ["documentation"]
+projects: ["scribe-org/2"]
+body:
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Terms
+      options:
+        - label: I have searched [documentation issues](https://github.com/scribe-org/Scribe-Android/issues?q=is%3Aissue+label%3Adocumentation)
+          required: true
+        - label: I agree to follow Scribe-Android's [Code of Conduct](https://github.com/scribe-org/Scribe-Android/blob/main/.github/CODE_OF_CONDUCT.md)
+          required: true
+  - type: input
+    id: topic
+    attributes:
+      label: Documentation Topic
+      placeholder: Briefly describe the documentation area you're working on.
+    validations:
+      required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      placeholder: Provide details of your contribution.
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        🔗 Please follow our [Documentation Guidelines](https://github.com/scribe-org/Scribe-Android/wiki/Documentation-Guidelines).
```

Comment:
I don't understand why a link to the GitHub wiki for this project is being included when this project doesn't have a wiki 🤔